hw/ip/otbn/doc/_index.md

```diff
@@ -374,6 +374,48 @@ Reads never stall.
         The accumulator register used by the {{< otbnInsnRef "BN.MULQACC" >}} instruction.
       </td>
     </tr>
+    <tr>
+      <td>0x4</td>
+      <td>RO</td>
+      <td>KEY_S0_L</td>
+      <td>
+Bits [255:0] of share 0 of the 384b OTBN sideload key provided by the [Key Manager]({{< relref "/hw/ip/keymgr/doc" >}}).
+
+A `KEY_INVALID` software error is raised on read if the Key Manager has not provided a valid key.
```

Comment:
I'm not sure about "valid key" here. It sounds like we're checking whether the key is valid or not. Maybe just "provided a key"?